install/operator/deploy/syndesis-operator.yml

```diff
@@ -125,6 +146,23 @@ subjects:
 roleRef:
   name: edit
 ---
+kind: ClusterRoleBinding
```

Comment:
@rhuss this is currently not present on any cluster, right? And thinking this further, we lack privileges to add new cluster level roles, so we have to run this by the OSO team.

Reply:
It's a hard requirement for this feature...

Reply:
Now I drilled down, sorry that I couldn't comment on this earlier. I really feel we must not have a cluster role binding required to install the operator.

This might change with the use of the OLM, but now this puts us in even more trouble (like the later fixes for multiple installations with creating multi-cluster roles, one per project, is a hackish workaround. That kind of cluster role really feels like a plain role, which is project dependent.

Personally, if I would have the choice and I would have to select between this feature (if it is not realizable differently) and requiring cluster role permissions, I would forgo the feature (its about not having to grant permissions as user, right ?)

Reply:
And I really, really apologize to not being able to make the review in a timely manner. shame on me. Just meeting/travel/whatnot overload. sorry.